CHANGELOG.md

```diff
@@ -0,0 +1,116 @@
+# 4.15.0 Macho Man (2024-11-14)
+
+## Features
+
+### Foreman Proxy Content
+ * Granular content counting on the UI ([#37945](https://projects.theforeman.org/issues/37945), [aadc393a](https://github.com/Katello/katello.git/commit/aadc393af67dee8d0b087d10e104c09f22c0e605))
+
+### Repositories
+ * Support on-demand for file repos ([#37929](https://projects.theforeman.org/issues/37929), [c32d9acd](https://github.com/Katello/katello.git/commit/c32d9acd6b8b681038720f897824f0ea5d43b644))
```

Comment:
Redmine was a bit messed up -- this one did not make Katello 4.15. You can just delete it.

Reply:
I've fixed redmine for the issue so it should not pop up again.